swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory else []
+    
+    def add(self, item):
+        self.inventory.append(item)
```

Comment:
What happens if we try to add an item to a vendor's inventory that is already in there?

Reply:
for the way our code is written, it is going to add it again?